src/instructlab/data/generate.py

```diff
@@ -194,6 +194,9 @@ def generate(
         if not api_base:
             api_base = ctx.obj.config.serve.api_base()
     try:
+        backends.check_model(
+            model, api_base, endpoint_url, http_client(ctx.params), ctx
+        )
```

Comment:
This isn't really a function of the model serving backend. At this point in the code, we may or may not be using `ilab` to run the model server.